node/src/app/cli.rs

```diff
@@ -173,8 +172,7 @@ impl Cli {
                 // initializer2_runner.run(&mut rng).await;
 
                 match initializer_runner.run(&mut rng).await {
-                    ReactorExit::ProcessShouldExit(ExitCode::Success) => return Ok(()),
-                    ReactorExit::ProcessShouldExit(exit_code) => process::exit(exit_code as i32),
+                    ReactorExit::ProcessShouldExit(exit_code) => return Ok(exit_code as i32),
```

Comment:
Are we going to log the exit code anywhere? If not, then maybe add it to the `cli.rs`.

Reply:
Done in [ad76d63](https://github.com/CasperLabs/casper-node/pull/1125/commits/ad76d638d8701e54d764214ec0bcba7463f8134f).